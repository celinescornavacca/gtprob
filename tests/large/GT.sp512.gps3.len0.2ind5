(((((((((((((XG0,(XG1,XG2)),((YG1,YG2),YG0)),(((RG1,RG2),RG0),((SG1,SG2),SG0))),((((CM0,(CM1,CM2)),(((HM1,HM2),HM0),(IM0,(IM1,IM2)))),(GL0,(GL1,GL2))),(EH0,(EH1,EH2)))),((((MO1,MO2),MO0),(((KP1,KP2),KP0),(LP0,(LP1,LP2)))),((DE0,(DE1,DE2)),((((ET1,ET2),ET0),(FT0,(FT1,FT2))),(WE0,(WE1,WE2)))))),((((((KC1,KC2),KC0),((((LO0,(LO1,LO2)),((GS0,(GS1,GS2)),(HS0,(HS1,HS2)))),(((RN1,RN2),RN0),((SN1,SN2),SN0))),(YE0,(YE1,YE2)))),((((RP1,RP2),RP0),((KQ0,(KQ1,KQ2)),((LQ1,LQ2),LQ0))),((DD1,DD2),DD0))),(((FG0,(FG1,FG2)),((GG1,GG2),GG0)),((((BM1,BM2),BM0),(((DR1,DR2),DR0),((ER1,ER2),ER0))),(((NC1,NC2),NC0),((AJ0,(AJ1,AJ2)),(BJ0,(BJ1,BJ2))))))),((((GC1,GC2),GC0),((HC1,HC2),HC0)),((VB1,VB2),VB0)))),(((NA0,(NA1,NA2)),((UL0,(UL1,UL2)),((VL1,VL2),VL0))),(((PM0,(PM1,PM2)),((QM1,QM2),QM0)),((IA1,IA2),IA0)))),((((AD0,(AD1,AD2)),((((RO0,(RO1,RO2)),((YS0,(YS1,YS2)),(ZS0,(ZS1,ZS2)))),(AN0,(AN1,AN2))),((WG1,WG2),WG0))),((WC0,(WC1,WC2)),((XC1,XC2),XC0))),((((((((UJ1,UJ2),UJ0),(VJ0,(VJ1,VJ2))),(RH0,(RH1,RH2))),(((TJ1,TJ2),TJ0),(((ZK0,(ZK1,ZK2)),((MS0,(MS1,MS2)),(NS0,(NS1,NS2)))),(((SK1,SK2),SK0),((TK1,TK2),TK0))))),(((QH1,QH2),QH0),((MN0,(MN1,MN2)),(NN0,(NN1,NN2))))),(((AF0,(AF1,AF2)),(((YM1,YM2),YM0),(ZM0,(ZM1,ZM2)))),(((SL1,SL2),SL0),((TL1,TL2),TL0)))),((((MB1,MB2),MB0),(((OL1,OL2),OL0),((PL1,PL2),PL0))),((((RD1,RD2),RD0),(((IP1,IP2),IP0),((JP1,JP2),JP0))),((((((((CS1,CS2),CS0),(DS0,(DS1,DS2))),((AL1,AL2),AL0)),(KK0,(KK1,KK2))),((GK1,GK2),GK0)),((UK0,(UK1,UK2)),(VK0,(VK1,VK2)))),(CD0,(CD1,CD2)))))))),(((BA0,(BA1,BA2)),((((((ID0,(ID1,ID2)),(((AG1,AG2),AG0),(BG0,(BG1,BG2)))),(((((YP0,(YP1,YP2)),((ZP1,ZP2),ZP0)),((FM0,(FM1,FM2)),((GM1,GM2),GM0))),(((AE0,(AE1,AE2)),(((NM0,(NM1,NM2)),((OM1,OM2),OM0)),((SH0,(SH1,SH2)),((FO0,(FO1,FO2)),(GO0,(GO1,GO2)))))),(((BP0,(BP1,BP2)),(((TR1,TR2),TR0),((AT0,(AT1,AT2)),((BT1,BT2),BT0)))),((ES0,(ES1,ES2)),(FS0,(FS1,FS2)))))),((((RB1,RB2),RB0),((((WN0,(WN1,WN2)),(((VQ0,(VQ1,VQ2)),((WQ1,WQ2),WQ0)),((QP1,QP2),QP0))),(MD0,(MD1,MD2))),((((((OP1,OP2),OP0),(PP0,(PP1,PP2))),(BL0,(BL1,BL2))),((TE1,TE2),TE0)),(((MP0,(MP1,MP2)),((NP1,NP2),NP0)),(XE0,(XE1,XE2)))))),(((YR0,(YR1,YR2)),(ZR0,(ZR1,ZR2))),((FF0,(FF1,FF2)),(GF0,(GF1,GF2))))))),((((RE0,(RE1,RE2)),((SE1,SE2),SE0)),((KI0,(KI1,KI2)),((LI1,LI2),LI0))),((ZA1,ZA2),ZA0))),((((JN0,(JN1,JN2)),((QT0,(QT1,QT2)),((RT1,RT2),RT0))),((ZC1,ZC2),ZC0)),(((EN1,EN2),EN0),((FN1,FN2),FN0)))),(((((QQ0,(QQ1,QQ2)),((RQ1,RQ2),RQ0)),((HL0,(HL1,HL2)),(IL0,(IL1,IL2)))),(((OF1,OF2),OF0),((OH0,(OH1,OH2)),(((((KL1,KL2),KL0),(LL0,(LL1,LL2))),((LK1,LK2),LK0)),(HK0,(HK1,HK2)))))),(((((SD1,SD2),SD0),((SQ0,(SQ1,SQ2)),((AS0,(AS1,AS2)),(BS0,(BS1,BS2))))),((XF0,(XF1,XF2)),((((US1,US2),US0),((VS1,VS2),VS0)),((IS0,(IS1,IS2)),((JS1,JS2),JS0))))),((ED1,ED2),ED0))))),(((((((ZL1,ZL2),ZL0),((AM1,AM2),AM0)),(((EB0,(EB1,EB2)),(((JJ0,(JJ1,JJ2)),(KJ0,(KJ1,KJ2))),(((UD0,(UD1,UD2)),(VD0,(VD1,VD2))),(((NO0,(NO1,NO2)),(OO0,(OO1,OO2))),(((LR0,(LR1,LR2)),((MR1,MR2),MR0)),(MQ0,(MQ1,MQ2))))))),((DG0,(DG1,DG2)),(EG0,(EG1,EG2))))),((((CB0,(CB1,CB2)),(((HB1,HB2),HB0),((CL0,(CL1,CL2)),((DL1,DL2),DL0)))),((SI0,(SI1,SI2)),((TI1,TI2),TI0))),((VA1,VA2),VA0))),(((((BD0,(BD1,BD2)),(((EO1,EO2),EO0),(((XO1,XO2),XO0),(YO0,(YO1,YO2))))),(((TD0,(TD1,TD2)),(((UE1,UE2),UE0),(((CT1,CT2),CT0),((DT1,DT2),DT0)))),((PE0,(PE1,PE2)),(QE0,(QE1,QE2))))),(((RA1,RA2),RA0),((RI0,(RI1,RI2)),((XL0,(XL1,XL2)),(YL0,(YL1,YL2)))))),((((ZJ1,ZJ2),ZJ0),(((TM1,TM2),TM0),(UM0,(UM1,UM2)))),(((((XI1,XI2),XI0),((YI1,YI2),YI0)),(((JR1,JR2),JR0),((KR1,KR2),KR0))),(((((ND1,ND2),ND0),(((CQ0,(CQ1,CQ2)),(DQ0,(DQ1,DQ2))),(((MI1,MI2),MI0),((NI1,NI2),NI0)))),(((FJ1,FJ2),FJ0),(GJ0,(GJ1,GJ2)))),(((KE0,(KE1,KE2)),(((((IT1,IT2),IT0),(JT0,(JT1,JT2))),((RM0,(RM1,RM2)),(SM0,(SM1,SM2)))),(((HR1,HR2),HR0),((IR1,IR2),IR0)))),((RC1,RC2),RC0))))))),((((ZG0,(ZG1,ZG2)),((AH1,AH2),AH0)),(((AK1,AK2),AK0),(BK0,(BK1,BK2)))),((WB1,WB2),WB0))))),((((((((CP0,(CP1,CP2)),((DP1,DP2),DP0)),((WL1,WL2),WL0)),((UC0,(UC1,UC2)),((VC1,VC2),VC0))),(((((((NE1,NE2),NE0),((OE1,OE2),OE0)),((TF0,(TF1,TF2)),(UF0,(UF1,UF2)))),((EJ0,(EJ1,EJ2)),(((TO1,TO2),TO0),((UO1,UO2),UO0)))),(((GH1,GH2),GH0),(HH0,(HH1,HH2)))),(((FB1,FB2),FB0),(((((AI1,AI2),AI0),(BI0,(BI1,BI2))),(((BN1,BN2),BN0),((QS0,(QS1,QS2)),(RS0,(RS1,RS2))))),(PB0,(PB1,PB2)))))),((FA1,FA2),FA0)),((((((((((((JO0,(JO1,JO2)),((KO1,KO2),KO0)),((IK1,IK2),IK0)),(IH0,(IH1,IH2))),(PF0,(PF1,PF2))),((DM0,(DM1,DM2)),((EM1,EM2),EM0))),(BC0,(BC1,BC2))),((XA1,XA2),XA0)),(((OB1,OB2),OB0),(((ZE0,(ZE1,ZE2)),(((LG1,LG2),LG0),(MG0,(MG1,MG2)))),(((NF1,NF2),NF0),((EL0,(EL1,EL2)),((FL1,FL2),FL0)))))),((((FD0,(FD1,FD2)),(((CE1,CE2),CE0),(((NQ0,(NQ1,NQ2)),((KT0,(KT1,KT2)),(LT0,(LT1,LT2)))),(XM0,(XM1,XM2))))),(((DB1,DB2),DB0),(((UR0,(UR1,UR2)),(VR0,(VR1,VR2))),(WM0,(WM1,WM2))))),((LD0,(LD1,LD2)),((WH0,(WH1,WH2)),(XH0,(XH1,XH2)))))),(((HG0,(HG1,HG2)),((IG1,IG2),IG0)),(HD0,(HD1,HD2)))),((AA1,AA2),AA0))),(((((JD0,(JD1,JD2)),(KD0,(KD1,KD2))),((((WA1,WA2),WA0),((((((((QI0,(QI1,QI2)),(((LM1,LM2),LM0),(((GT0,(GT1,GT2)),((HT1,HT2),HT0)),(QN0,(QN1,QN2))))),(VG0,(VG1,VG2))),((((ZN1,ZN2),ZN0),(AO0,(AO1,AO2))),(JK0,(JK1,JK2)))),(((PK0,(PK1,PK2)),(((DO1,DO2),DO0),(((UP1,UP2),UP0),((VP1,VP2),VP0)))),(VE0,(VE1,VE2)))),((KS0,(KS1,KS2)),((LS1,LS2),LS0))),((AB1,AB2),AB0)),((JB0,(JB1,JB2)),(KB0,(KB1,KB2))))),(QA0,(QA1,QA2)))),(((YD0,(YD1,YD2)),((((ZF1,ZF2),ZF0),((((BR0,(BR1,BR2)),((CR1,CR2),CR0)),((IQ0,(IQ1,IQ2)),((JQ1,JQ2),JQ0))),(((XQ1,XQ2),XQ0),((YQ1,YQ2),YQ0)))),((FE1,FE2),FE0))),((II0,(II1,II2)),(JI0,(JI1,JI2))))),(((WP0,(WP1,WP2)),((XP1,XP2),XP0)),((CA1,CA2),CA0))))),((((((((LB1,LB2),LB0),(((OD1,OD2),OD0),((TN0,(TN1,TN2)),((UN1,UN2),UN0)))),((((OS1,OS2),OS0),(PS0,(PS1,PS2))),(((BF0,(BF1,BF2)),(CF0,(CF1,CF2))),(((CI1,CI2),CI0),((DI1,DI2),DI0))))),((((QL0,(QL1,QL2)),((RL1,RL2),RL0)),(YJ0,(YJ1,YJ2))),(SA0,(SA1,SA2)))),(((HO0,(HO1,HO2)),(IO0,(IO1,IO2))),((LA1,LA2),LA0))),(((IB1,IB2),IB0),((((HF1,HF2),HF0),(((YF0,(YF1,YF2)),(((QK1,QK2),QK0),((RK1,RK2),RK0))),(((ML0,(ML1,ML2)),(NL0,(NL1,NL2))),(NH0,(NH1,NH2))))),(((XN1,XN2),XN0),((YN1,YN2),YN0))))),((HA0,(HA1,HA2)),(((AC0,(AC1,AC2)),((((PJ1,PJ2),PJ0),((QJ1,QJ2),QJ0)),(((PH1,PH2),PH0),((CK0,(CK1,CK2)),(DK0,(DK1,DK2)))))),(((FI0,(FI1,FI2)),((GI0,(GI1,GI2)),(HI0,(HI1,HI2)))),((VH1,VH2),VH0)))))),((((((YB1,YB2),YB0),((((FH0,(FH1,FH2)),(((KN0,(KN1,KN2)),(LN0,(LN1,LN2))),(ZI0,(ZI1,ZI2)))),((DF0,(DF1,DF2)),((EF1,EF2),EF0))),((EC1,EC2),EC0))),(((((BH0,(BH1,BH2)),((CH1,CH2),CH0)),(((DH0,(DH1,DH2)),(((RJ1,RJ2),RJ0),(SJ0,(SJ1,SJ2)))),((KF1,KF2),KF0))),((TA1,TA2),TA0)),(((HJ1,HJ2),HJ0),((IJ1,IJ2),IJ0)))),(((((LC0,(LC1,LC2)),(MC0,(MC1,MC2))),(((WD1,WD2),WD0),(((VF0,(VF1,VF2)),((WF1,WF2),WF0)),(GE0,(GE1,GE2))))),(((TB0,(TB1,TB2)),((UB1,UB2),UB0)),((UA1,UA2),UA0))),(((GA0,(GA1,GA2)),(((OI0,(OI1,OI2)),((PO0,(PO1,PO2)),(QO0,(QO1,QO2)))),(((MH0,(MH1,MH2)),((JM0,(JM1,JM2)),(((GN1,GN2),GN0),(HN0,(HN1,HN2))))),(((KH1,KH2),KH0),(LH0,(LH1,LH2)))))),(((((((GB1,GB2),GB0),((((TQ0,(TQ1,TQ2)),(UQ0,(UQ1,UQ2))),(PI0,(PI1,PI2))),((XB1,XB2),XB0))),(((IC1,IC2),IC0),((JC1,JC2),JC0))),(MA0,(MA1,MA2))),((((((GP1,GP2),GP0),((HP1,HP2),HP0)),(KM0,(KM1,KM2))),(SF0,(SF1,SF2))),((((((ZD1,ZD2),ZD0),(((MK1,MK2),MK0),(((VM1,VM2),VM0),((SS0,(SS1,SS2)),(TS0,(TS1,TS2)))))),((((FR1,FR2),FR0),((GR1,GR2),GR0)),(((YH1,YH2),YH0),(ZH0,(ZH1,ZH2))))),(SB0,(SB1,SB2))),(QB0,(QB1,QB2))))),((JA0,(JA1,JA2)),((((JL1,JL2),JL0),((SP0,(SP1,SP2)),((TP1,TP2),TP0))),((XD1,XD2),XD0))))))),((((((((ZO1,ZO2),ZO0),((AP1,AP2),AP0)),((IN1,IN2),IN0)),(FC0,(FC1,FC2))),((((SO1,SO2),SO0),(((GQ1,GQ2),GQ0),((HQ1,HQ2),HQ0))),((WK0,(WK1,WK2)),((XK1,XK2),XK0)))),(((((LF1,LF2),LF0),(MF0,(MF1,MF2))),(((RR1,RR2),RR0),((SR1,SR2),SR0))),((KA1,KA2),KA0))),(((((((JH1,JH2),JH0),(((WJ1,WJ2),WJ0),(XJ0,(XJ1,XJ2)))),((((((TH0,(TH1,TH2)),((UH1,UH2),UH0)),(((ON1,ON2),ON0),((PN1,PN2),PN0))),(((EK0,(EK1,EK2)),((FK1,FK2),FK0)),((VN0,(VN1,VN2)),(((EQ1,EQ2),EQ0),(FQ0,(FQ1,FQ2)))))),((((NJ0,(NJ1,NJ2)),((OJ1,OJ2),OJ0)),(((NK1,NK2),NK0),(OK0,(OK1,OK2)))),((EE1,EE2),EE0))),(((((CG0,(CG1,CG2)),(((NG1,NG2),NG0),((OG1,OG2),OG0))),(((YC1,YC2),YC0),(((UI1,UI2),UI0),(((CN0,(CN1,CN2)),(DN0,(DN1,DN2))),((MM1,MM2),MM0))))),(((AQ0,(AQ1,AQ2)),(BQ0,(BQ1,BQ2))),(OC0,(OC1,OC2)))),(((((CJ1,CJ2),CJ0),((DJ1,DJ2),DJ0)),(GD0,(GD1,GD2))),(NB0,(NB1,NB2)))))),(((((CC1,CC2),CC0),(DC0,(DC1,DC2))),(((QF0,(QF1,QF2)),((RF1,RF2),RF0)),(BE0,(BE1,BE2)))),((((((((VO1,VO2),VO0),(WO0,(WO1,WO2))),(((PR1,PR2),PR0),(QR0,(QR1,QR2)))),(((PG1,PG2),PG0),(QG0,(QG1,QG2)))),(((VI0,(VI1,VI2)),((WI1,WI2),WI0)),(((MT1,MT2),MT0),(NT0,(NT1,NT2))))),(((IF1,IF2),IF0),(JF0,(JF1,JF2)))),(BB0,(BB1,BB2))))),(EA0,(EA1,EA2))),(((WR1,WR2),WR0),(XR0,(XR1,XR2))))))),((((SC1,SC2),SC0),(TC0,(TC1,TC2))),(((((((ZB0,(ZB1,ZB2)),(((WS0,(WS1,WS2)),(XS0,(XS1,XS2))),((JE1,JE2),JE0))),((JG0,(JG1,JG2)),(KG0,(KG1,KG2)))),((DA0,(DA1,DA2)),((((((((LJ1,LJ2),LJ0),(MJ0,(MJ1,MJ2))),(PD0,(PD1,PD2))),(((LE1,LE2),LE0),(ME0,(ME1,ME2)))),((QD0,(QD1,QD2)),((ZQ0,(ZQ1,ZQ2)),((AR1,AR2),AR0)))),((BO0,(BO1,BO2)),(CO0,(CO1,CO2)))),((((OQ1,OQ2),OQ0),(PQ0,(PQ1,PQ2))),((YK1,YK2),YK0))))),(((PC1,PC2),PC0),((QC1,QC2),QC0))),((((HE1,HE2),HE0),((IE1,IE2),IE0)),((YA0,(YA1,YA2)),(((EI1,EI2),EI0),((OT0,(OT1,OT2)),(PT0,(PT1,PT2))))))),(((((OA1,OA2),OA0),(PA0,(PA1,PA2))),((((TG1,TG2),TG0),(UG0,(UG1,UG2))),((NR0,(NR1,NR2)),((OR1,OR2),OR0)))),(((EP1,EP2),EP0),((FP1,FP2),FP0))))));
