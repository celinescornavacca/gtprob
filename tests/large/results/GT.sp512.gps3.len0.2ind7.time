72211.03user 231.71system 22:23:55elapsed 89%CPU (0avgtext+0avgdata 1999724maxresident)k
204inputs+0outputs (0major+44025769minor)pagefaults 0swaps
