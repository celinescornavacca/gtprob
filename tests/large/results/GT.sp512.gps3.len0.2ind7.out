((((((((CB1,CB2